service/file_transaction_service.py

```diff
@@ -30,7 +32,7 @@ def GetStatus(self, request, context):
 
             # Check connection to Azure Blob Storage
             try:
-                utility.block_blob_service.get_blob_service_properties()
+                azure_client.block_blob_service.get_blob_service_properties()
             except:
```

Comment:
Pep 8 style guide error
Consult https://stackoverflow.com/questions/30442236/how-to-prevent-too-broad-exception-in-this-case